examples/trap_test.go

```diff
@@ -1,41 +0,0 @@
-package examples
```

Comment:
this was definitely not an `example` but an e2e test, and is a dup of the e2e test used in jit previously. So I deleted it